### DISCUSSION TITLE
Breaking Changes

### DISCUSSION THREAD

**Discussion by FredericLatour** [axios/axios#5351] [Q&A]
Hi guys,
I can't see any breaking changes documented.
This is something that was already asked in this discussion #4761 and was never answered.
How can I possibly get the breaking changes if I would like to move from 0.27.2 to the latest 1.2.x version.
Am I missing something ?
Thanks

**Comment by ForgeSeanTyler** [comment]
More discussion on this topic in the thread here: https://github.com/axios/axios/discussions/4996
